placeSpecificContent/theSpot/moneyTree/entityScripts/moneyTreeGiverClient.js

```diff
@@ -0,0 +1,205 @@
+// moneyTreeClient.js
+
+//  Created by Mark Brosche on 10-18-2018
+//  Copyright 2018 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+/* global EventBridge Users AccountServices Agent Avatar */
+
+(function (){
+// get userData
+// spawn Overlays for clients with ID == giverID
+// delete after overlay after click event or after timer expires.
+// send data to google sheet
+    var COINS_AVAILABLE = SoundCache.getSound(Script.resolvePath('../resources/sounds/chimes-loop.wav')),
+        COIN_CLICKED = SoundCache.getSound(Script.resolvePath('../resources/sounds/coin-click.wav')),
+        PAYOUT = SoundCache.getSound(Script.resolvePath('../resources/sounds/payout.wav')),
+        SECRETS = Script.require(Script.resolvePath('../moneyTreeURLs.json')),
```

Comment:
this is dangerous because it is running on a client script accessible to anyone. hopefully not a huge deal.